main.py

```diff
@@ -67,40 +67,51 @@ def main():
                 for sid in ROSTERS[course_id]:
                     print_information(f"- {sid}: {STUDENTS[sid]}")
 
+                
                 #remove the cast to an int, to check if its an empty string
                 student_id = input("Enter Student ID: ")
                 while(student_id == ""):
                     print("You must enter a student id! ")
                     student_id = input("Enter Student ID: ")
 
-                #cast the string back into an int
-                student_id = int (student_id)
-
-
-                isGradeEmpty = True
-                while (isGradeEmpty):
-                    grade = input("Enter Grade: ") 
-
-                    if (not grade or grade == "" or grade.startswith(" ")):
-                        print("\tGrade cannot be empty")
-                        continue
-                        
-                    else: 
-                        isGradeEmpty = False
-
+                #trys to cast the string back into an int
                 try:
-                    grade_value = float(grade)
-                    if grade_value < 0:
-                        print_error("Grade cannot be negative.")
-                        input("Press enter to continue.")
-                        continue  # Go back to menu
+                    student_id = int (student_id)
                 except ValueError:
-                    print_error("Invalid grade format. Please enter a number.")
-                    input("Press enter to continue.")
-                    continue  # Go back to menu
+                        print_error("Invalid ID format. Please enter a number.")
+                        continue  # Go back to menu
```

Comment:
The indentation is wrong. Also, the error message won't show. You can run main.py and double check that.

Reply:
Probably have the same issue with other error messages.